sdk/textanalytics/azure-ai-textanalytics/samples/async_samples/sample_authentication_async.py

```diff
@@ -39,11 +39,11 @@ class AuthenticationSampleAsync(object):
     async def authentication_with_subscription_key_async(self):
```

Comment:
do you think changing the name here is important? maybe `authentication_with_api_key_async(self)`

Reply:
Good catch! Changed the name. Do you mind approving again?